### ISSUE TITLE
Interpolation of the text could be done by nesting a text inside of text block

### ISSUE THREAD

**Issue by tomatuxtemple** [viewstools/morph#56]
<img width="354" alt="screen shot 2018-02-06 at 10 02 07 pm" src="https://user-images.githubusercontent.com/3190986/35884143-f33cd1ac-0b89-11e8-9c9d-865a02aace5a.png">

**Comment by dariocravero** [comment]
We've been discussing this with @tomatuxtemple and I think we got to a more resilient solution for it that would work with I18n too going forward! Would love to hear your thoughts on this @amymc.

Say you want to have `Greeting` text that says `Hi, Amy!`, using the text inside text model, we can:

```
Greeting Text
text Hi, Name!
Name Text
text <name
color red
```

There are a few things happening there. First the `Name` text block is inside the `Greeting`, it doesn't have to be an inline block, it could also be a view that we just use here, like:
```
Greeting Text
text Hi, Name!
Name
text <name
```

Then, because the block is named `Name`, we can now link it within `Greeting`'s `text` prop by just writing it!

The nice thing about this pattern is that it is extensible to support cases like when we don't know the name, eg:
```
Greeting Text
text Hi, Name!
when <!name
text Hi stranger!
Name
text <name
```
or
```
Greeting Text
text Hi, Name!
Name
text <name
when <!name
text randomer
```

**Comment by dariocravero** [comment]
Implemented in #85! We should add the docs now.
